README.md

```diff
@@ -24,6 +25,18 @@ The Teleporter protocol, on the other hand, is implemented at the smart contract
 - [Docs](#docs)
 - [Resources](#resources)
 
+## Deployed Addresses
+
+| Contract              | Address                                        | Chain                    |
+| --------------------- | ---------------------------------------------- | ------------------------ |
+| `TeleporterMessenger` | **0x253b2784c75e510dD0fF1da844684a1aC0aa5fcf** | All chains, all networks |
+| `TeleporterRegistry`  | **0x7C43605E14F391720e1b37E49C78C4b03A488d98** | Mainnet C-Chain          |
+| `TeleporterRegistry`  | **0xf86cb19ad8405aefa7d09c778215d2cb6ebfb228** | Fuji C-Chain             |
+
+- `TeleporterMessenger` is deployed to the same address on all chains using Nick's method. See [Teleporter Contract Deployment](./utils/contract-deployment/README.md) and [Deploy Teleporter to a Subnet](#deploy-teleporter-to-a-subnet) for more details.
```

Comment:
Can we add something that explains that it's not _automatically_ deployed, but once it is, it'll always be at the same address?

Reply:
Good call. How about rephrasing slightly to "When deployed, `TeleporterMessenger` will end up at the same address..."